index.php

```diff
@@ -27,6 +27,75 @@ function gutenberg_menu() {
 }
 add_action( 'admin_menu', 'gutenberg_menu' );
 
+
+$registered_blocks = array();
+
+/**
+ * Registers a block.
+ *
+ * @param  string   $name   Block name including namespace.
+ * @param  function $render Render function called to generate the HTML output
+ */
+function register_block( $name, $render ) {
+	global $registered_blocks;
+	$registered_blocks[ $name ] = $render;
```

Comment:
I don't want to preemptively assume requirements, but do you think we'd need a block to register options aside from just a render?